scrum/tools/sprint_math.py

```diff
@@ -265,7 +299,7 @@ def calc_current_sprint(self):
         self.sp_retro_total = self.sp_retro_completed + self.sp_retro_leftover
 
         # Calculate the next Sprint
-        self.calc_planned_next_sprint()
+        self.sp_next_sprint = self.calc_planned_next_sprint()
```

Comment:
does this need to be printed or returned?

Reply:
Changed it so that it is returned and assigned in the init method